docs/getting-started/installation/intellij.md

```diff
@@ -0,0 +1,6 @@
+# IntelliJ IDEA
+Rather than VS Code, when developing with kotlin the reccomended IDE is IntelliJ. IntelliJ offers much better autocomplete and syntax highlighting when using Kotlin and Java together (which we always are, as WPILib is Java)
+
+You'll want to download the latest version of the "Community" edition, this is the free version. Installing this is pretty standard, but if you have any issues there are tons of resources online for it.
```

Comment:
I think we should also try avoiding saying 'you'll want'. Just another tone thing. 

It might be better to just explain: There are two versions of IntelliJ IDEA, Ultimate and Community. Community is free to download, but if you already have access to Ultimate, you can use that for FRC development too.

> Installing this is pretty standard

Again, let's avoid assuming prior knowledge. Link to install guides or add screenshots